spec.html

```diff
@@ -5900,10 +5901,10 @@ <h1>CreateArrayFromList ( _elements_ )</h1>
       <p>The abstract operation CreateArrayFromList takes argument _elements_ (a List). It is used to create an Array object whose elements are provided by _elements_. It performs the following steps when called:</p>
       <emu-alg>
         1. Assert: _elements_ is a List whose elements are all ECMAScript language values.
-        1. Let _array_ be ! ArrayCreate(0).
+        1. Let _array_ be ! ArrayCreate(0<sub>𝔽</sub>).
         1. Let _n_ be 0.
         1. For each element _e_ of _elements_, do
-          1. Perform ! CreateDataPropertyOrThrow(_array_, ! ToString(_n_), _e_).
+          1. Perform ! CreateDataPropertyOrThrow(_array_, ! ToString(𝔽(_n_)), _e_).
```

Comment:
Maybe it would be better to use:

```
1. Let _nNumber_ be the Number value for _n)_.
1. Perform ! CreateDataPropertyOrThrow(_array_, ! ToString(_nNumber_), _e_).
```

This pattern is already used in other places of this PR, so it would be good to be consistent.

Reply:
I prefer the more terse form. Is there a benefit to splitting it out?

Reply:
For this case there is no much benefit, but there are other places where we keep applying `𝔽(_n_)` very often. The advantage of splitting them out is that it is easier for someone not involved with ECMA262 understand, while `𝔽(_n_)` requires understanding what `𝔽` means. I'm fine with leaving it as it is. What do you think of other places where we are using splitted version?